controller/state.go

```diff
@@ -81,6 +81,14 @@ type comparisonResult struct {
 	timings map[string]time.Duration
 }
 
+func (res *comparisonResult) GetSyncStatus() *v1alpha1.SyncStatus {
+	return res.syncStatus
+}
+
+func (res *comparisonResult) GetHealthStatus() *v1alpha1.HealthStatus {
+	return res.healthStatus
+}
+
```

Comment:
Can these please have simple tests, so that we don't unnecessarily decrease test coverage?